src/services/price/index.ts

```diff
@@ -0,0 +1,15 @@
+import { PriceServiceError } from "@domain/price"
+import { getCurrentPrice } from "@services/realtime-price"
+
+export const PriceService = (): IPriceService => {
+  const fetchPrice = async () => {
+    const price = await getCurrentPrice()
```

Comment:
this method can throw exceptions, please use try/catch and return PriceServiceError